components/ide/jetbrains/gateway-plugin/doc/installing.md

```diff
@@ -1,6 +0,0 @@
-## Installing Gitpod plugin in JetBrains Gateway
```

Comment:
let's move it to a separate PR, we can remove it only after we deployed other changes